.github/workflows/main.yml

```diff
@@ -13,3 +13,12 @@ jobs:
         run: |
           npm install
           make test
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run ESLint
+        run: |
+          npm install
+          make eslint
```

Comment:
Maybe we can run both eslint and tests in one job?

Reply:
@lazycoder9 What's the point of it? 
I see many pros of separated jobs:
1. We have different results for different checks. (broken tests and lints separately)
2. We run jobs in parallel

Reply:
@npupko maybe it is not so bad thing, but I thought about running `npm install` twice

Reply:
@npupko anyway, we can accept it now, because it is not a big deal. Approved

Reply:
@lazycoder9 Yeah, I have thought about it but don't found any solution for now.
GitLab has `before_script` section and `stage: build` for it.